### PR TITLE
Add json format output to project get

### PR DESCRIPTION
**Pull request by jichenjc**
**What type of PR is this?**
> Uncomment only one ` /kind` line, and delete the rest.
> For example, `> /kind bug` would simply become: `/kind bug`

 /kind feature
**What does does this PR do / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #?

**PR acceptance criteria**:

- [ ] Unit test 

- [ ] Integration test 

- [ ] Documentation 

**How to test changes / Special notes to the reviewer**:
`odo project get -o json`

### CODE REVIEW

**Comment by mik-dass** [comment]
Please add a integration test for it.

File: tests/integration/project/cmd_project_test.go [hunk 1]
```diff
@@ -49,17 +49,16 @@ var _ = Describe("odo project command tests", func() {
...
+	Context("odo machine readable output on empty project", func() {
+		It("should be able to list current project", func() {
+			projectListJSON := helper.CmdShouldPass("odo", "project", "list", "-o", "json")
+			listOutputJSON, err := helper.Unindented(projectListJSON)
+			Expect(err).Should(BeNil())
+			partOfProjectListJSON, err := helper.Unindented(`{"kind":"Project","apiVersion":"odo.dev/v1alpha1","metadata":{"name":"` + project + `","namespace":"` + project + `","creationTimestamp":null},"spec":{},"status":{"active":true}}`)
+			Expect(err).Should(BeNil())
+			Expect(listOutputJSON).To(ContainSubstring(partOfProjectListJSON))
+		})
+	})
```
Comment (mik-dass) [inline tests/integration/project/cmd_project_test.go:61]: The issue mentioned is not resolved yet. This will lead to flakes.

**Comment by mik-dass** [comment]
@jichenjc Why did we close this?

**Comment by girishramnani** [comment]
The PR got automatically closed because a force push removed all the code and made code diff 0

**Comment by girishramnani** [comment]
Use git reflog to get the removed commits @jichenjc

**Comment by jichenjc** [comment]
ok, weird .. will get it back , thanks for the reminder

**Comment by jichenjc** [comment]
https://github.com/openshift/odo/pull/3712 is submitted on behalf of wrong close here

**Pull request closed without merging**